pillar/journal-public.sls

```diff
@@ -20,15 +20,18 @@ journal:
             User-Agent: *
             Disallow: $robots_disallow
             Disallow: /download/
+        # regarding trailing slashes:
+        # - https://developers.google.com/search/reference/robots_txt#url-matching-based-on-path-values
+        # it shouldn't make a difference but turnitinbot thinks it does
```

Comment:
Which bit are you looking at? I see:

<img width="709" alt="Screenshot 2020-02-18 at 08 36 46" src="https://user-images.githubusercontent.com/1784740/74718303-df32b300-5229-11ea-8a00-2296278d9f81.png">

Where it does make a difference (which makes sense, as they are different paths).

Reply:
these bits:

![Screenshot at 2020-02-20 09-49-08](https://user-images.githubusercontent.com/2142748/74885929-b562c300-53c6-11ea-8434-cbda8cdf71ae.png)
![Screenshot at 2020-02-20 09-52-01](https://user-images.githubusercontent.com/2142748/74885930-b693f000-53c6-11ea-94da-9b1672a3a624.png)
![Screenshot at 2020-02-20 09-52-15](https://user-images.githubusercontent.com/2142748/74885931-b72c8680-53c6-11ea-9fbb-dafd9879ae93.png)

see this:
![Screenshot at 2020-02-20 09-53-36](https://user-images.githubusercontent.com/2142748/74886023-ef33c980-53c6-11ea-8693-d6712bfe73ee.png)

My understanding all this time has been that these robots.txt rules were to block the search results from being scraped. Was that correct? 

I've only just now looked at how we do pagination with the search results (assuming it was `/search/something`) and it's actually `/search?something`, which means the turnitbot guys are right and I was wrong - blocking `/search/` would *not* have included `/search`, which is what we're after (if my understanding of our intent is correct).

Reply:
That’s my understanding too, so the change is right. Let’s drop the comment though, since that isn’t.

Reply:
done